tests/runner.py

```diff
@@ -1041,9 +1041,12 @@ def do_run(self, src, expected_output, force_c=False, **kwargs):
         f.write(src)
     self._build_and_run(filename, expected_output, **kwargs)
 
+  def do_runf(self, filename, expected_output, **kwargs):
```

Comment:
this could perhaps have a better name, but I don't have a particularly good idea, and I like that it's nice and short (like `btest`).

Reply:
Yeah.. we sadly have 4 different flavors of `do_run` now... I would welcome a better naming scheme but I don't have one in mind right now.